packages/app-bridge/src/AppBridgeBlock.ts

```diff
@@ -98,8 +102,16 @@ export interface AppBridgeBlock<
         callback: EventCallbackParameter<EventName, Event>,
     ): EventUnsubscribeFunction;
 
+    /**
+     * @deprecated This will be removed in version 4.0.0 of `@frontify/app-bridge`
+     * Use `appBridge.context('blockId').get()` instead
```

Comment:
nit for this and following cases:
```suggestion
     * Use `appBridge.context('blockId').get()` instead.
```